src/components/SearchBar/SearchBar.component.tsx

```diff
@@ -11,15 +12,21 @@ interface SearchBarProps {
     setSearchQuery: (query: string) => void;
 }
 
-function SearchBar({ setFilter, setSort, setSearchQuery }: SearchBarProps) {
-    const [selectedFilters, setSelectedFilters] = useState<string[]>([]);
+const categories: Readonly<Pick<Category, 'id' | 'name'>[]> = [
+    { id: 1, name: 'Clothes' },
+    { id: 2, name: 'Electronics' },
+    { id: 3, name: 'Furniture' },
+    { id: 4, name: 'Shoes' },
+    { id: 5, name: 'Miscellaneous' },
+];
+
+export function SearchBar({ setFilter, setSort, setSearchQuery }: SearchBarProps) {
+    const [selectedFilter, setSelectedFilter] = useState<string>('');
     const [searchInput, setSearchInput] = useState<string>('');
 
-    const handleFilterChange = (filter: string) => {
-        setSelectedFilters((previousFilters) =>
-            previousFilters.includes(filter) ? previousFilters.filter((f) => f !== filter) : [...previousFilters, filter],
-        );
-        setFilter(filter);
+    const handleFilterChange = (filterID: string) => {
+        setSelectedFilter(filterID);
+        return setFilter(filterID);
     };
 
     const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
<img width="1440" alt="Screenshot 2024-07-06 at 18 48 57" src="https://github.com/VasylBiliak/fe-react-2024/assets/115080771/4a7d29f1-d296-4b02-b3e3-b0a5826c1d6a">

when I write in lowercase name of product I don't see it. Need to fix it. Need to compare names in lowercase

Reply:
I've investigated the issue where searching for product names in lowercase doesn't yield results. It turns out that the backend is case-sensitive when handling product name searches. To resolve this, the backend should ideally normalize the case of the search input and product names to ensure case-insensitive searching.
![Screenshot_1](https://github.com/VasylBiliak/fe-react-2024/assets/147659216/dbd61246-4ed4-4033-acd4-a309a8d868fe)

Reply:
![Screenshot_2](https://github.com/VasylBiliak/fe-react-2024/assets/147659216/4ca20051-3f32-45cd-9756-8131cefd0008)